test/incompatible.jl

```diff
@@ -0,0 +1,22 @@
+function test_incompatible()
+  solvers = Dict(:lbfgs => [:unc],
+                 :trunk => [:unc],
+                 :tron  => [:unc, :bnd])
+  problems = Dict(:unc => ADNLPModel(x -> 0, zeros(2)),
+                  :bnd => ADNLPModel(x -> 0, zeros(2), lvar=zeros(2), uvar=ones(2)),
+                  :equ => ADNLPModel(x -> 0, zeros(2), c=x->[0.0], lcon=[0.0], ucon=[0.0]),
+                  :ine => ADNLPModel(x -> 0, zeros(2), c=x->[0.0], lcon=[-Inf], ucon=[0.0]),
+                  :gen => ADNLPModel(x -> 0, zeros(2), c=x->[0.0; 0.0], lcon=[-Inf; 0.0], ucon=zeros(2))
+                 )
+  for (ptype, problem) in problems
+    @testset "Testing that solver that don't handle type $ptype raise error: " begin
```

Comment:
Move this line into the next `for` and use `"Testing that $solver ..."`

Reply:
Changed